docusaurus/docs/operate/configs/supplier_staking_config.md

```diff
@@ -98,15 +98,16 @@ For example, it must match the regex `^[a-zA-Z0-9_-]{1,8}$`, and spaces are disa
 _`Required`_, _`Non-empty`_
 
 `endpoints` is a list of `endpoint` objects that the `Supplier` will advertise
-to the Pocket Network. Each `endpoint` object consists of an `url` and a `rpc_type`.
+to the Pocket Network. Each `endpoint` object consists of a `publicly_exposed_url`
+and a `rpc_type`.
 
-##### `url`
+##### `publicly_exposed_url`
 
 _`Required`_
 
-The `url` defines the endpoint for sending `RelayRequests` from the Pocket Network's `Gateways` and `Applications`. Typically, this endpoint is provided by a RelayMiner, which acts as a proxy. This setup means that the specified URL directs to the RelayMiner's endpoint, which in turn, routes the requests to the designated service node.
+The `publicly_exposed_url` defines the endpoint for sending `RelayRequests` from the Pocket Network's `Gateways` and `Applications`. Typically, this endpoint is provided by a RelayMiner, which acts as a proxy. This setup means that the specified URL directs to the RelayMiner's endpoint, which in turn, routes the requests to the designated service node.
```

Comment:
`provided by a RelayMiner, which acts as a proxy. This setup means that the specified URL directs to the RelayMiner's endpoint,`

It's a bit confusing that "it's provided by a RelayMiner` and then "directs to a RelayMine`.

Could you try rewording this a bit?